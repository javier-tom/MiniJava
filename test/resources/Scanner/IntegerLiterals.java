12345678910
1
200
51
0
245
123
