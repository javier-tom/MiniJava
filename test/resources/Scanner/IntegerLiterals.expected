INTEGER_LITERAL(12345678910) INTEGER_LITERAL(1) INTEGER_LITERAL(200) INTEGER_LITERAL(51) INTEGER_LITERAL(0) INTEGER_LITERAL(245) INTEGER_LITERAL(123)